apps/dashboard/app/models/batch_connect/app.rb

```diff
@@ -145,6 +145,14 @@ def ssh_allow?
       form_config[:ssh_allow]
     end
 
+    def tooltip_html
+      settings = recent_settings
+      if !settings.empty?
+        html = '<b> Parameters </b><hr class="thin">'
+        html += settings.join '<br>'
+      end
+    end
```

Comment:
This is probably better as a view or a controller helper method rather than here in the model.